courses/B.yaml

```
@@ -26,7 +26,11 @@
   desc: |
     LaTeX is the standard method for making publications in the
     computational and physical sciences.
-
+  reading: |
+    Short LaTeX `tutorial <http://andre-geldenhuis.github.io/Latex-Short/>`_ based on Software Carpentry 
```

The github link one seems a bit unfinished, and git history shows it was last updated in 2015.  Maybe we could find a newer one?
```suggestion
    Short LaTeX `tutorial <http://andre-geldenhuis.github.io/Latex-Short/>`_ based on Software Carpentry 
    Latex: a document preparation system, by Leslie Lamport (offline book, canonical reference)
```
That book above is the "canonical" reference, but it's not online...  I've seen this one way back when I was learning, maybe it's the second best?: https://tobi.oetiker.ch/lshort/lshort.pdf